Formula/docker-slim.rb

```diff
@@ -25,7 +25,7 @@ class DockerSlim < Formula
   skip_clean "bin/slim-sensor"
 
   def install
-    ENV["CGO_ENABLED"] = "0"
+    system "go", "generate", "./pkg/appbom"
```

Comment:
- relates to https://github.com/slimtoolkit/slim/commit/5f45991c81dacb749b4329b9a7b77732e39c38dc